packages/eas-cli/src/commands/branch/publish.ts

```diff
@@ -107,13 +137,19 @@ export default class BranchPublish extends Command {
       description: `return a json with the new update group.`,
       default: false,
     }),
+    auto: flags.boolean({
```

Comment:
maybe consider sth more descriptive, meaning of the auto in context of publishing is not really obvious

`--auto-branch`/`--auto-message`
`--managed`
or use `--non-interactive` flag and if values are not provided via `--message`/`--branch` us those defaults

Also if you are considering adding updates config to eas.json, options like this would be a good candidate for it

Reply:
The issue with `--non-interactive` is that is not clear that this means "apply an opinionated default" instead of "throw if essential args are not provided"

`--auto-branch/--auto-message` are nice, but maybe too verbose?

> Also if you are considering adding updates config to eas.json, options like this would be a good candidate for it

Interesting idea, I'm a bit hesitant to engage there right now as it is in such flux that things tend to get reverted. Maybe once things get more established?

@jonsamp curious what you think about these other flag suggestions.

Reply:
Thanks for the discussion, will go with `--auto` for now!